config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :users, only: [:create]
```

Comment:
I like the use of resources here.  I will need to go back on my branch and do this.